src/main/kotlin/org/sert2521/bunnybots2019/drivetrain/Drivetrain.kt

```diff
@@ -8,14 +8,30 @@ class Drivetrain : Subsystem("Drivetrain", ::controlDrivetrain) {
     private val right = MotorController(MotorControllers.rightFront, MotorControllers.rightBack) {
         inverted = true
```

Comment:
What does inverted mean? Why is it true?

Reply:
Good question! The value of `inverted` (true or false) determines which way the motor is spinning. Changing it from one to the other is a really quick way to change the direction a motor is going. 

For example, when we first pushed the flicky things code, everything was running in the wrong direction. By setting `inverted` to true, we can quickly get everything spinning the way we want it to. 

I believe it's set true there because that's how it was when we tested it. It may be different now. Before this is merged in, I'll change the value to what we have in the `development` branch.